Dockerfile

```diff
@@ -79,7 +80,7 @@ RUN set -eux; \
     sudo apt-get -y clean; \
     sudo rm -rf /var/lib/apt/lists/*; \
     pip3 install awscli; \
-    pip3 install requests requests-unixsocket; \
+    pip3 install --force-reinstall "urllib3<2" requests requests-unixsocket; \
```

Comment:
`urllib` is install from `awscli` too:
```
2023-10-11T09:08:42.1804281Z #55 32.15 Collecting urllib3<2.1,>=1.25.4
2023-10-11T09:08:42.1804876Z #55 32.16   Downloading urllib3-2.0.6-py3-none-any.whl (123 kB)
```
```
2023-10-11T09:08:45.2914183Z #55 35.31 Successfully installed PyYAML-6.0.1 awscli-1.29.62 botocore-1.31.62 colorama-0.4.4 docutils-0.16 jmespath-1.0.1 pyasn1-0.5.0 python-dateutil-2.8.2 rsa-4.7.2 s3transfer-0.7.0 urllib3-2.0.6
```
Should not we downgrading it first at install then?

Reply:
Thanks. I've moved the downgraded install earlier to not install twice.